### ISSUE TITLE
Missing docs for ndef emulation 

### ISSUE THREAD

**Issue by CyberGl1tch** [gassajor000/pn532pi#24]
Is it possible to use this Python library on a Raspberry Pi to perform NFC card emulation, specifically to emulate a forum tag v2 or v4 on Android?

**Comment by gassajor000** [comment]
Yes that capability should be supported. The example for ndef emulation hasn't been ported to python yet but you can look at it to attempt to convert it yourself. https://github.com/gassajor000/pn532pi/blob/master/examples/emulate_tag_ndef/emulate_tag_ndef.ino
